src/components/AnswerCard.js

```diff
@@ -55,8 +56,12 @@ function AnswerCard ({ answer: answerNode }) {
   })
 
   function vote (value) {
+    const newInput = (viewerVote.edges.length > 0 && value === viewerVote.edges[0].node.credible)
```

Comment:
In addition to removing the relay connection (`edges`) after changes to the backend, I think this code would be better suited inside of the `massageInput` function which can be passed to `useForm`. For example, starting on line 37:

```javascript
...
  } = useForm({
    mutation,
    defaultInput: {
      answerId: id,
      credible: null
    },
    massageInput: input => {
      newInput = Object.assign({}, input)
      if (viewerVote && value === viewerVote.credible) delete newInput.value
      return newInput
    }
  })
...
```

Reply:
fyi, not your fault for not knowing that option existed. `useForm` is a hand coded React hook without any documentation aside from the code itself.